keyboards/handwired/frame_48/frame_48.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2022 Greg Mathews
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+#define LAYOUT_frame_48(L00, L01, L02, L03, L04, L05, R00, R01, R02, R03, R04, R05, R06, L10, L11, L12, L13, L14, L15, L16, R10, R11, R12, R13, R14, R15, R16, L20, L21, L22, L23, L24, L25, L26, R20, R21, R22, R23, R24, R25, R26, L30, L31, L32, L33, L34, L35, R30, R31, R32, R33, R34, R35, R36) \
+    {                                                                                                                                                                                                                                                                                                               \
+        {L00, L01, L02, L03, L04, L05}, \
+        {L10, L11, L12, L13, L14, L15, L16}, \
+        {L20, L21, L22, L23, L24, L25, L26}, \
+        {L30, L31, L32, L33, L34, L35}, \
+        {KC_NO, R01, R02, R03, R04, R05, R06}, \
+        {R10, R11, R12, R13, R14, R15, R16}, \
+        {R20, R21, R22, R23, R24, R25, R26}, \
+        {KC_NO, R31, R32, R33, R34, R35, R36}                                                                                                                                                                                                                                                                                                           \
+    }
```

Comment:
I fail to see any difference between `LAYOUT` and `LAYOUT_frame_48`? If there is none, one of the layouts should be removed. In which case, `LAYOUT` would be more favorable in terms of naming conventions.